nuxeo-runtime/nuxeo-connect-standalone/src/test/java/org/nuxeo/launcher/connect/TestConnectBroker.java

```diff
@@ -328,544 +363,678 @@ public void testInstallLocalPackageRequest() throws Exception {
+                + "Added G-1.0.1-SNAPSHOT\n" //
+                + "Download of 'H-1.0.1-SNAPSHOT' will replace the one already in local cache.\n" //
+                + "Downloading [H-1.0.1-SNAPSHOT]...\n" //
+                + "Replacement of H-1.0.1-SNAPSHOT in local cache...\n" //
+                + "Added H-1.0.1-SNAPSHOT\n" //
+                + "\n" //
+                + "As package 'J-1.0.1' has an optional dependency on package(s) [G-1.0.1-SNAPSHOT, H-1.0.1-SNAPSHOT] currently being installed, it will be reinstalled.\n" //
+                + "Dependency resolution:\n" //
+                + "  Installation order (3):        G-1.0.1-SNAPSHOT/H-1.0.1-SNAPSHOT/J-1.0.1\n" //
+                + "  Uninstallation order (1):      J-1.0.1\n" //
+                + "  Unchanged packages (7):        A:1.0.0, B:1.0.1-SNAPSHOT, hfA:1.0.0, C:1.0.0, D:1.0.2-SNAPSHOT, studioA:1.0.0, K:1.0.0-SNAPSHOT\n" //
+                + "  Local packages to install (3): G:1.0.1-SNAPSHOT, H:1.0.1-SNAPSHOT, J:1.0.1\n" //
+                + "  Local packages to remove (1):  J:1.0.1\n" //
+                + "\n" //
+                + "Uninstalling J-1.0.1\n" //
+                + "Installing G-1.0.1-SNAPSHOT\n" //
+                + "Installing H-1.0.1-SNAPSHOT\n" //
+                + "Installing J-1.0.1";
+        assertThat(logOf(logCaptureResult)).isEqualTo(expectedLogs);
     }
 
     @Test
     @LogCaptureFeature.FilterWith(PkgRequestLogFilter.class)
     public void testUninstallPackageRequestWithOptionalDependencies() throws Exception {
-        Environment.getDefault().setProperty(Environment.DISTRIBUTION_NAME, "server");
-        Environment.getDefault().setProperty(Environment.DISTRIBUTION_VERSION, "8.3");
-        ConnectBroker connectBrocker = new ConnectBroker(Environment.getDefault());
-        ((StandaloneCallbackHolder) NuxeoConnectClient.getCallBackHolder()).setTestMode(true);
-        connectBrocker.setAllowSNAPSHOT(false);
+        connectBroker.setAllowSNAPSHOT(false);
 
         // Before: [studioA-1.0.0, hfA-1.0.0, A-1.0.0, B-1.0.1-SNAPSHOT, C-1.0.0, D-1.0.2-SNAPSHOT, G-1.0.1-SNAPSHOT,
         // H-1.0.1-SNAPSHOT, J-1.0.1]
-        checkPackagesState(connectBrocker, Arrays.asList("studioA-1.0.0", "hfA-1.0.0", "A-1.0.0", "B-1.0.1-SNAPSHOT",
+        checkPackagesState(connectBroker, Arrays.asList("studioA-1.0.0", "hfA-1.0.0", "A-1.0.0", "B-1.0.1-SNAPSHOT",
                 "C-1.0.0", "D-1.0.2-SNAPSHOT", "G-1.0.1-SNAPSHOT", "H-1.0.1-SNAPSHOT", "J-1.0.1"),
                 PackageState.STARTED);
-        checkPackagesState(connectBrocker,
+        checkPackagesState(connectBroker,
                 Arrays.asList("studioA-1.0.1", "studioA-1.0.2-SNAPSHOT", "hfB-1.0.0", "hfC-1.0.0-SNAPSHOT", "A-1.2.0",
                         "A-1.2.1-SNAPSHOT", "A-1.2.2-SNAPSHOT", "A-1.2.2", "A-1.2.3-SNAPSHOT", "B-1.0.1", "B-1.0.2",
                         "C-1.0.1-SNAPSHOT", "C-1.0.2-SNAPSHOT", "D-1.0.3-SNAPSHOT", "D-1.0.4-SNAPSHOT"
```

Comment:
I'm doubtful regarding the utility of the method. Anyway, keep at least the same (meaningful) name.
=> `checkPackagesState(PackageState expectedState, String... packageIds)`